generated/broker_1/roster_6.json

```diff
@@ -117,9 +117,7 @@
             "plan_name": "KP DC Silver 2000/35",
             "plan_type": "HMO",
             "metal_level": "Silver",
-            "benefit_group_name": "FULL-TIME EMPLOYEES",
-            "terminated_on": "2017-03-10",
-            "terminate_reason": "I have coverage through an individual market health plan"
```

Comment:
Fixed it. 

https://github.com/dchealthlink/HBX-mobile-app-APIs/commit/0f001b344659822c1fc94e3812363b800c75aa26